site/components/navigator/style.css

```diff
@@ -55,10 +55,13 @@
 }
 
 .item {
+  background-color: transparent;
+  border: none;
   width: 36px;
```

Comment:
Good shout! Could we just increase the height of the buttons a bit so that they're easier to tap/click? The tap area is quite small vertically by default

![Screenshot 2019-04-30 at 15 19 17](https://user-images.githubusercontent.com/5824792/56968676-86d7db00-6b5b-11e9-8fb1-271fbdb9d4c8.png)


```suggestion
   width: 36px;
   height: 36px;
```

Reply:
Nice catch. I've made them the same height as the arrows now.